definitions/divorce/json/CaseEventToFields.json

```diff
@@ -6161,7 +6293,9 @@
     "PageLabel": "Grant Decree Nisi",
     "PageDisplayOrder": 1,
     "FieldShowCondition": "DecreeNisiGranted=\"Yes\" AND DivorceCostsOptionDN != \"endClaim\" AND D8DivorceCostsClaim=\"Y*\"",
-    "ShowSummaryChangeOption": "Yes"
+    "ShowSummaryChangeOption": "Yes",
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/sol-dn-review-petition",
+    "RetriesTimeoutURLMidEvent": "120,120"
```

Comment:
Not sure this should be here. /sol-dn-review-petition is used to generate a mini petition link, isn't it?

Reply:
@hosainnet